src/audio/drc/drc_math_generic.c

```diff
@@ -226,11 +226,11 @@ inline int32_t drc_inv_fixed(int32_t x, int32_t precision_x, int32_t precision_y
 inline int32_t drc_sin_fixed(int32_t x)
 {
 	const int32_t PI_OVER_TWO = Q_CONVERT_FLOAT(1.57079632679489661923f, 30);
```

Comment:
Remove "f" from this constant end to get bet better precision. The conversion is handled by C preprocessor so there's no double precision floats code injected.